examples/petsc/qfunctions/area/areacube.h

```diff
@@ -109,7 +109,7 @@ CEED_QFUNCTION(SetupMassGeoCube)(void *ctx, const CeedInt Q,
 // -----------------------------------------------------------------------------
 
 // *****************************************************************************
-// This QFunction applies the mass matrix for a scalar field.
+// This QFunction applies the mass operator for a scalar field.
```

Comment:
Would we want to apply this change elsewhere?

Reply:
I'm updating it in all examples